dashboard/test/models/activity_test.rb

```diff
@@ -1,5 +1,6 @@
 require 'test_helper'
 
+require 'async_progress_handler'
```

Comment:
does this need to be required here? I think it should already be required by the rails app since it's eventually referenced by what you're testing here.